TextInputKit/TextInputKit/Code/TextInputFormat/TextInputFormat+Formatter.swift

```diff
@@ -10,6 +10,15 @@ import Foundation
 
 extension TextInputFormat {
 
+    /// Creates a `Formatter` with the object type `FormatterObjectValue<Value>`.
+    ///
+    /// The `FormatterObjectValue<Value>` is used to wrap a `Value` if present, or a string if no value is deserizlied from that string.
+    /// The reason to use a `FormatterObjectValue<Value>` wrapper is the following `Formatter` and `NSTextField` behavior:
```

Comment:
It would be nice if you leave a note that this is a system behaviour. Now it looks like you framework do so.